akka-samples/akka-sample-persistence-scala/build.sbt

```diff
@@ -6,5 +6,7 @@ scalaVersion := "2.11.7"
 
 libraryDependencies ++= Seq(
   "com.typesafe.akka" %% "akka-actor" % "2.4-SNAPSHOT",
-  "com.typesafe.akka" %% "akka-persistence" % "2.4-SNAPSHOT"
```

Comment:
akka-actor was removed when we had the build failure, I have added it again
stange that akka-sample-persistence-java works without that dependency